README.md

```diff
@@ -359,6 +359,14 @@ Determines if fields in the changeset with `nil` values will be inserted as Dyna
 
 Determines if, during **Repo.update** or **Repo.update_all**, fields in the changeset with `nil` values will be removed from the record/s or set to the DynamoDB `null` value. This option is also available inline per query.
 
+**:empty_mapset_to_nil** :: boolean, *default:* `false`
+
+When a field contains a `DynamoDBSet` type and has a MapSet in it, by default the mapset must contain one or more values, otherwise an `insert` on the object will fail. This is due to DynamoDB not supporting empty sets. Setting this value to `true` will convert empty mapset values to `nil` before writting, allowing them to be written to DynamoDB as a null value.
```

Comment:
Should "writting" be "writing"?

Reply:
Sure, if you want to spell things the "traditional" way.